include/loop_tool/measure.hpp

```diff
@@ -0,0 +1,311 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
```

Comment:
this is in libsysml now right?

Reply:
Not yet, I also just copied that file as well, since I didn't know how to include things from a different place from include folder.